Sources/SwiftTemplate.swift

```diff
@@ -3,9 +3,16 @@
 // Copyright (c) 2017 SwiftGen
 // MIT Licence
 //
-
+import Foundation
 import Stencil
 
+#if os(Linux)
+#if swift(>=3.1)
```

Comment:
Dunno, Rakefile (Xcode project) and swift package test the same thing. Testing both would just double the length of the CI tests.